<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="horizontalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea">
      <property name="background">
       <brush brushstyle="SolidPattern">
        <color alpha="180">
         <red>160</red>
         <green>160</green>
         <blue>160</blue>
        </color>
       </brush>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_N"/>
    <addaction name="action_O"/>
    <addaction name="separator"/>
    <addaction name="action_S"/>
    <addaction name="action_A"/>
    <addaction name="separator"/>
    <addaction name="action_E"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_C"/>
    <addaction name="action_V"/>
    <addaction name="separator"/>
    <addaction name="action_BM"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>窗口(&amp;W)</string>
    </property>
    <addaction name="action_CW"/>
    <addaction name="separator"/>
    <addaction name="action_T"/>
    <addaction name="action_L"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_N">
   <property name="text">
    <string>新建文件(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_O">
   <property name="text">
    <string>打开文件(&amp;O)</string>
   </property>
  </action>
  <action name="action_S">
   <property name="text">
    <string>保存文件(&amp;S)</string>
   </property>
  </action>
  <action name="action_A">
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
  </action>
  <action name="action_E">
   <property name="text">
    <string>退出(&amp;E)</string>
   </property>
  </action>
  <action name="action_Z">
   <property name="text">
    <string>撤销(&amp;Z)</string>
   </property>
  </action>
  <action name="action_C">
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
  </action>
  <action name="action_V">
   <property name="text">
    <string>粘贴(&amp;V)</string>
   </property>
  </action>
  <action name="action_BM">
   <property name="text">
    <string>编码格式</string>
   </property>
  </action>
  <action name="action_CW">
   <property name="text">
    <string>关闭窗口(&amp;C)</string>
   </property>
  </action>
  <action name="action_T">
   <property name="text">
    <string>窗口平铺(&amp;T)</string>
   </property>
  </action>
  <action name="action_L">
   <property name="text">
    <string>窗口叠放(&amp;L)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
